example/scripts/lock-stack.js

```diff
@@ -18,41 +18,30 @@ const aws = require('@cumulus/common/aws');
 const LOCK_TABLE_NAME = 'cumulus-int-test-lock';
 const STACK_EXPIRATION_MS = 120 * 60 * 1000; // 2 hours
 
-async function performLock(mutex, deployment, cb) {
-  try {
-    await mutex.writeLock(deployment, STACK_EXPIRATION_MS);
-
-    return cb(0);
-  } catch (e) {
-    return cb(1);
-  }
+function performLock(mutex, deployment) {
+  return mutex.writeLock(deployment, STACK_EXPIRATION_MS);
 }
 
-async function removeLock(mutex, deployment, cb) {
-  try {
-    await mutex.unlock(deployment);
-
-    return cb(0);
-  } catch (e) {
-    return cb(1);
-  }
+function removeLock(mutex, deployment) {
+  return mutex.unlock(deployment);
 }
 
-async function updateLock(lockFile, deployment, cb) {
+function updateLock(lockFile, deployment) {
   const dynamodbDocClient = aws.dynamodbDocClient({
     convertEmptyValues: true
   });
-
   const mutex = new concurrency.Mutex(dynamodbDocClient, LOCK_TABLE_NAME);
 
   if (lockFile === 'true') {
-    return performLock(mutex, deployment, cb);
+    return performLock(mutex, deployment);
   }
-
-  return removeLock(mutex, deployment, cb);
+  return removeLock(mutex, deployment);
 }
 
 // Assuming this is run as:
 // node lock-stack.js (true|false) deployment-name
 // true to lock, false to unlock
-updateLock(process.argv[2], process.argv[3], console.log);
+updateLock(process.argv[2], process.argv[3]).catch((e) => {
+  console.log(e);
```

Comment:
By removing the catches in the function above, if the stack is already locked the error will come down here and output some ugly output to the console. So I think we can probably just remove this `console.log`. Or we could check the error type.

It's this every 30 seconds:
```
{ ConditionalCheckFailedException: The conditional request failed
    at Request.extractError (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/protocol/json.js:51:27)
    at Request.callListeners (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/request.js:683:14)
    at Request.transition (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/request.js:685:12)
    at Request.callListeners (/Users/lpfreder/workspace/cumulus/packages/common/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
  message: 'The conditional request failed',
  code: 'ConditionalCheckFailedException',
  time: 2019-05-09T18:33:43.271Z,
  requestId: 'F9DS660P03DB8F05JV5BRH345BVV4KQNSO5AEMVJF66Q9ASUAAJG',
  statusCode: 400,
  retryable: false,
  retryDelay: 14.034934931016307 }
```

Reply:
Hmm, good point.    I think I want to check the error rather than just swallow it.  I'll issue a fix.

Reply:
2698f8755